mmdt/survey/migrations/0001_initial.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.2.4 on 2024-02-25 12:25
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SurveyForm',
+            fields=[
+                ('form_name', models.CharField(default='DefaultFormName', max_length=255)),
+                ('form_id', models.AutoField(primary_key=True, serialize=False)),
+                ('is_active', models.BooleanField(default=False)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='SurveyQuestion',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('question_name', models.CharField(max_length=255)),
+                ('pub_date', models.DateTimeField(verbose_name='date published')),
+                ('is_enabled', models.BooleanField(default=True)),
+                ('survey_group', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='questions', to='survey.surveyform')),
```

Comment:
[Just confirmation]
Does this mean SurveyQuestion and SurveyForm linked with this survey_group field?? which one shall we use as as key from SurveyForm to link with survey_group in SurveyQuestion?

Reply:
Yes ama, I found out that Django ORM understands and can automatically use the primary key field `form_id` of the referenced model `SurveyForm` as a foreign key for `survey_group` field from the `SurveyQuestion` model although I did not define which field to use a a primary key.